src/Exclusion/ExpressionLanguageExclusionStrategy.php

```diff
@@ -31,6 +32,33 @@ public function __construct(ExpressionEvaluatorInterface $expressionEvaluator)
         $this->expressionEvaluator = $expressionEvaluator;
     }
 
+
+    /**
+     * {@inheritDoc}
+     */
+    public function shouldSkipClass(ClassMetadata $class, Context $navigatorContext): bool
+    {
+        if (null === $class->excludeIf) {
+            return false;
+        }
+
+        $variables = [
+            'context' => $navigatorContext,
+            'property_metadata' => $class,
```

Comment:
I believe that should be called class_metadata

Reply:
Yepp!